### ISSUE TITLE
Can not reproduce office-31 and officehome dataset results

### ISSUE THREAD

**Issue by Millielele** [ispc-lab/BMD#1]
Hi, thanks for your excellent work. 
I can't reproduce the results based on the office-31 and officehome dataset. I have changed the parameter S, α=0.3，β=0.1 according to the article. Are there another parameters I need to change? How can I solve that?

**Comment by sanqingqu** [comment]
Sorry for the late reply. Thanks for your interest in our BMD.

Similar to SHOT, on Office-31 and Office-Home, you need to change the loss function to:
```
if args.dataset == "Office" or args.dataset == "OfficeHome":
   loss = ent_loss + 0.3 * psd_loss + 0.1 * dym_loss - reg_loss

elif args.dataset == "VisDA":
   if epoch_idx >= 1.0:
      loss = ent_loss + 2.0 * psd_loss + 0.5 * dym_loss
   else:
      loss = ent_loss - reg_loss
 ```
